recipes/fastgltf/all/conanfile.py

```diff
@@ -23,11 +23,15 @@ class fastgltf(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "enable_small_vector": [True, False],
+        "disable_custom_memory_pool": [True, False],
+        "use_64bit_float": [True, False],
```

Comment:
There options are not available for 0.4 and 0.5:

- https://github.com/spnda/fastgltf/blob/v0.6.0/CMakeLists.txt#L16
- https://github.com/spnda/fastgltf/blob/v0.5.0/CMakeLists.txt#L15
- https://github.com/spnda/fastgltf/blob/v0.4.0/CMakeLists.txt#L15

Please, remove them in the `config_options` like:

```
if Version(self.version) <= "0.6.0":
    del self.options.disable_custom_memory_pool
    del self.options.use_64bit_float
```

Then, you will need to get those options using safe method:

```
if self.options.get_safe("disable_custom_memory_pool"):
    tc.variables["FASTGLTF_DISABLE_CUSTOM_MEMORY_POOL"] = True
```